doc/nrf/libraries/bluetooth_services/mesh/gen_loc_cli.rst

```diff
@@ -13,17 +13,98 @@ Unlike the Generic Location Server model, the Generic Location Client only creat
+
+``mdl_loc instance get-all``
+----------------------------
+
+	Print all instances of the Generic Location Client model on the device.
+
+
+``mdl_loc instance set <elem_idx>``
+-----------------------------------
+
+	Select the Generic Location Client model instantiated on the specified element id. This instance will be used in message sending. If no model instance was selected, the first model instance found on the device will be used by default.
+
+	* ``elem_idx``: Element index where to find the model instance.
+
+
+``mdl_loc global-get``
+----------------------
+
+	Get the current global location value of the Generic Location state.
+
+
+``mdl_loc global-set <latitude> <longitude> <altitude>``
+--------------------------------------------------------
+
+	Set a global location value of the Generic Location state and wait for a response.
+
+	* ``latitude``: Global WGS84 North coordinate in degrees.
+	* ``longitude``: Global WGS84 East coordinate in degrees.
+	* ``altitude``: Global altitude above the WGS84 datum in meters.
+
+
+``mdl_loc global-set-unack <latitude> <longitude> <altitude>``
+--------------------------------------------------------------
+
+	Set a global location value of the Generic Location state without requesting a response.
+
+	* ``latitude``: Global WGS84 North coordinate in degrees.
+	* ``longitude``: Global WGS84 East coordinate in degrees.
+	* ``altitude``: Global altitude above the WGS84 datum in meters.
+
+
+``mdl_loc local-get``
+---------------------
+
+	Get the current local location value of the Generic Location state.
+
+
+``mdl_loc local-set <north> <east> <altitude> <floor> [time_delta [precision_mm [is_mobile]]]``
+-----------------------------------------------------------------------------------------------
+
+	Set a global location value of the Generic Location state and wait for a response.
+
+	* ``north``: Local north position in decimeters.
+	* ``east``: Local east position in decimeters.
+	* ``altitude``: Local altitude in decimeters.
+	* ``floor``: Floor number.
+	* ``time_delta``: If present, defines time since the previous position update in milliseconds.
+	* ``precision_mm``: If present, defines precision of the location in millimeters.
+	* ``is_mobile``: If present, defines whether the device is movable.
+
+
+``mdl_loc local-set-unack <north> <east> <altitude> <floor> [time_delta [precision_mm [is_mobile]]]``
+-----------------------------------------------------------------------------------------------------
+
+	Set a local location value of the Generic Location state without requesting a response.
+
+	* ``north``: Local north position in decimeters.
+	* ``east``: Local east position in decimeters.
+	* ``altitude``: Local altitude in decimeters.
+	* ``floor``: Floor number.
+	* ``time_delta``: If present, defines time since the previous position update in milliseconds.
+	* ``precision_mm``: If present, defines precision of the location in millimeters.
```

Comment:
```suggestion
	* ``time_delta``: If present, defines the time since the previous position update in milliseconds.
	* ``precision_mm``: If present, defines the precision of the location in millimeters.
```